From a61806cc068e0eda14ae9529cc230ecdb01cb292 Mon Sep 17 00:00:00 2001
From: Dan Liang <dan.liang@intel.com>
Date: Tue, 30 Apr 2013 13:34:16 +0800
Subject: [PORT FROM MAIN] Media: Enable time lapse recording.

BZ: 47858

Original BZ is 23657

Change-Id: I553ea4c473e30770a18464d522222164028c4765
Orig-Change-Id: Ia1aaaaaf48dae47e562cc622c4d1a6ea01e8a9d9
Signed-off-by: Zhao Liang <leo.zhao@intel.com>
Signed-off-by: Dan Liang <dan.liang@intel.com>
---
 media/libstagefright/OMXCodec.cpp |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/media/libstagefright/OMXCodec.cpp b/media/libstagefright/OMXCodec.cpp
index b9ae503..fd1d19b 100644
--- a/media/libstagefright/OMXCodec.cpp
+++ b/media/libstagefright/OMXCodec.cpp
@@ -399,6 +399,10 @@ sp<MediaSource> OMXCodec::Create(
                     codec->mFlags |= kOnlySubmitOneInputBufferAtOneTime;
                 }
 
+                if (!strncmp("OMX.Intel.VideoEncoder", componentName, 22)) {
+                    codec->mFlags &= ~kOnlySubmitOneInputBufferAtOneTime;
+                }
+
                 return codec;
             }
 
-- 
1.7.4.1

